src/libs/API.js

```diff
@@ -122,30 +122,8 @@ function handleExpiredAuthToken(originalCommand, originalParameters, originalTyp
     isAuthenticating = true;
 
     // eslint-disable-next-line no-use-before-define
```

Comment:
Do we still need this?

Reply:
Yeah I think we do since `reauthenticate()` is still defined after this function.